swarm/network/stream/delivery.go

```diff
@@ -205,6 +208,11 @@ R:
 			continue R
 		default:
 		}
+		if d.validator != nil {
+			if !d.validator.Validate(&chunk.Key, chunk.SData) {
+				log.Error("Invalid chunk", "key", chunk.Key)
```

Comment:
this check has no consequence now. if invalid chunk should not be p;ut to localstore and the peer should be dropped